src/LoginPage.cpp

```diff
@@ -95,8 +95,6 @@ LoginPage::LoginPage(QWidget *parent)
              "address there, if your server doesn't support .well-known lookup.\nExample: "
              "@user:server.my\nIf Nheko fails to discover your homeserver, it will show you a "
              "field to enter the server manually."));
-        matrixid_input_->setValidator(
-          new QRegularExpressionValidator(QRegularExpression("@.+?:.{3,}"), this));
```

Comment:
This prevents the red underline from showing up, if the input is invalid. It would be cool, if you could re-add that somehow, but if it turns out to be too complicated, just tell me. What do you think?

Reply:
Honestly, I haven't got any ideas about it.

Reply:
The relevant code is in src/ui/TextField.cpp:149. Currently it uses hasAcceptableInput, which may need to be overridden.

Reply:
Okay, I will check it.

Reply:
Sure, if it turns out to be difficult, you don't need to do it, it would just be nice to have! <3

Reply:
Okay :)